app/Http/Controllers/AirlineController.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Airline;
+use App\Models\City;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use Illuminate\Validation\Rule;
+use Illuminate\View\View;
+
+class AirlineController extends Controller
+{
+    public function index(): View
+    {
+        return view('airlines.show', [
```

Comment:
I don't understand why you're passing the pagination links for Airline, and the cities? Could you provide more context?

Reply:
I want to take advantage of the pagination provided by laravel, so I thought of passing the links to the view in order to be able to use it there.

Reply:
Yeah but, you're just passing the pagination links and not the airline records themselves. Aren't you supposed to render the airlines in the table from `airlines.show`? You should include the airlines as part of the view data to do so:
```
return view('airlines.show', [
    'airlines' => Airline::paginate(),
]);
```